Makefile

```diff
@@ -101,3 +110,15 @@ gen-check: gen
 .PHONY: serve
 serve:
 	hack/local-docsite-up.sh
+
+PHONY: init-codegen
+init-codegen:
+	hack/init-codegen.sh
+
+.PHONY: gen-crd
+gen-crd: init-codegen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
+	$(CONTROLLER_GEN) rbac:roleName=manager-role crd webhook paths="./..." output:crd:artifacts:config=manifests/crd/bases
```

Comment:
manifests/crd/bases -> manifests/charts/base/templates

make it configurable

Reply:
completed